code/modules/admin/create_poll.dm

```diff
@@ -0,0 +1,122 @@
+/client/proc/create_poll()
+	set name = "Create Poll"
+	set category = "Special Verbs"
+	if(!check_rights(R_PERMISSIONS))	return
+	if(!dbcon.IsConnected())
+		src << "<span class='danger'>Failed to establish database connection.</span>"
+		return
+	var/polltype = input("Choose poll type.","Poll Type") in list("Single Option","Text Reply","Rating","Multiple Choice")
+	var/choice_amount = 0
+	switch(polltype)
+		if("Single Option")
+			polltype = "OPTION"
```

Comment:
I'm aware, but this is BYOND and SQL here.
I'll do touchups of poll's code like this later, probably in another PR.